TypeCobol/Compiler/Scopes/RootSymbolTable.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Compiler.Scopes
+{
+    /// <summary>
+    /// The Root Symbol Table is a special Namespace
+    /// </summary>
+    public class RootSymbolTable : NamespaceSymbol
+    {
+        public VariableSymbol BottomVariable { get; private set;}
```

Comment:
Ok we have a universe of variables, the BottomVariable can be seen as the variable 0, or the null variable of the universe.

Reply:
To make all other variables not use the index 0, if for instance in a bit set of symbols the bit 0 is set this will be probably means that the algorithm using such symbol index is wrong.